packages/query/src/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './__generated/graphql';
 export * from './query-engine';
+export * from './utils/query-generation';
+export type { TypedDocumentNode } from '@apollo/client';
```

Comment:
We _really_ shouldn't be exporting things from 3rd party libraries.

(and when possible we should be getting our types and graphql functions from our codegen folder)

Reply:
Codegen does not provide this type. Its supposed to be imported from client implementation. Or from a lib which every client uses '@graphql-typed-document-node/core'

Reply:
https://the-guild.dev/graphql/codegen/plugins/typescript/typed-document-node also the `graphql` import

Im not saying we need to rewrite our codegen rules altogether bur I do want to double check what client preset gives us.

Reply:
tl;dr this can be removed. the pattern is fine, but we're importing it directly in the one place we need it.